napari/_qt/dialogs/activity_dialog.py

```diff
@@ -109,6 +109,8 @@ def __init__(self, parent=None, toggle_button=None):
         self._baseLayout.addWidget(self._titleBar)
         self._baseLayout.addWidget(self._scrollArea)
         self.setLayout(self._baseLayout)
+        self.resize(520, self.MIN_HEIGHT)
```

Comment:
>520

Is that based on?

should there be a `MIN_WIDTH` ?

Reply:
the width of that widget was already a little oddly determined... depending on the construction order.  I couldn't find any rhyme or reason in the code, so I explicitly set it to the width that it was getting before this change

Reply:
Ok